tests/basic-manifest.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": ["context.jsonld", {"@base": "basic-manifest"}],
+  "@id": "",
+  "@type": "mf:Manifest",
+  "name": "Tests using YAML-LD JSON Profile",
+  "description": "These tests implement the requirements for the [YAML-LD JSON Profile](https://json-ld.github.io/yaml-ld/spec/#yaml-ld-json-profile).",
+  "baseIri": "https://json-ld.github.io/yaml-ld/tests/",
+  "sequence": [
+    {
+      "@id": "#cr-utf8-positive-1",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
+      "name": "UTF-8 BOM",
+      "purpose": "A YAML-LD document MUST be encoded in UTF-8, to ensure interoperability with [[JSON]].",
+      "input": "basic/cr-utf8-positive-1-in.yamlld",
+      "expect": "basic/cr-utf8-positive-1-out.yamlld"
+    },
+    {
+      "@id": "#cr-utf8-negative-1",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "UTF-6 BOM",
```

Comment:
I guess this is meant to be "UTF-16"?

Reply:
See also apparently related line 136 —
```
                Test cr-utf8-negative-1 UTF-6 BOM
```
— and line 142 —
```
                  <dt>Name</dt><dd>UTF-6 BOM</dd></dt>
```
— of tests/basic-manifest.html

Reply:
Those last two are derived from the .jsonld version.